code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrganizerEventStats.java

```diff
@@ -0,0 +1,110 @@
+package com.example.eventlinkqr;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+/** This activity is the page where organizers can track event statistics.
+ * In the current state it is very barebones and serves as a home for our map fragment.
+ * */
+public class OrganizerEventStats extends AppCompatActivity implements OnMapReadyCallback {
+
+    private GoogleMap myMap;
+
+    //This is a hardcoded location set around edmonton. Later we'll pull this from firebase
+    private List<LatLng> locations = generatePoints(new LatLng(53.5461, -113.4938), 30);
+
+    /**
+     * This is just chatGPT code to generate random points until Firebase is setup
```

Comment:
I know this is temporary, but it can't hurt to cite it. That way if we forget, we don't go to jail.